composer.json

```diff
@@ -43,12 +43,6 @@
           "phpunit",
           "@reset-ruleset"
         ],
-        "test-coverage": [
```

Comment:
This can be retained to conveniently run test coverage locally if needed.

Reply:
It doesn't work without other packages and another script. I'd rather just remove what doesn't work.

Reply:
ok